### ISSUE TITLE
CHF Module uitbouwen in Synthea

### ISSUE THREAD

**Issue by Grettnn** [dHealthNL/synthea-modules#3]
[CHF Module Synthea.pdf](https://github.com/dHealthNL/synthea-modules/files/4408809/CHF.Module.Synthea.pdf)

**Comment by ruudz** [comment]
![image](https://user-images.githubusercontent.com/2395361/78777657-3b989000-799a-11ea-8c81-c0a5883cd15b.png)
@Grettnn, dit hadden wij al eerder besproken maar ontgaat mij nu even. Zou jij hier nog een keer kunnen uitleggen waar `Attribute` op slaat in de splitsing van het eerste blok in bovenstaande plaatje? Dus `Attribute == acuut` en `Else: Attribute == geleidelijk`.

**Comment by Grettnn** [comment]
Hi @ruudz ! Eigenlijk wil ik dat er dan bij diagnose dat er acuut hartfalen komt te staan ipv hartfalen. Dit wordt dus in de diagnose (lees: mogelijk probleem) gespecificeerd. Dus ik denk dat het gewoon kan zijn If acuut hartfalen en aan de andere kant Else hartfalen (zonder acuut). Attribute kan dan eigenlijk weg.

**Comment by ruudz** [comment]
Ok, dus of het accuut is of niet wordt bepaald bij blokje `Symptomen HF`?

**Comment by Grettnn** [comment]
Ja, want ook voor die symptomen geldt (eigenlijk alleen dyspneu) of het acuut is ontstaan. Als de patiënt acuut dyspneu heeft 

EN zou je Oedeem nog als symptoom willen toevoegen. Ik zie nu dat dit blokje mist: 118654009 | aandoening gekenmerkt door oedeem (aandoening) |

**Comment by Grettnn** [comment]
Hey @ruudz !
Ik heb een typefout gemaakt in het diagrammetje:
Pro-BNP => 400 pg/ml = de NT Pro-BNP sorry!
![image](https://user-images.githubusercontent.com/62705289/85724799-d9c40880-b6f4-11ea-8527-dd85b5f5775f.png)

**Comment by Grettnn** [comment]
geldt ook voor de andere stroom, dus waar je Pro-BNP ziet staan moet gewoon NT Pro-BNP staan :)

**Comment by ruudz** [comment]
Ah, top! En de BNP waardes?

**Comment by Grettnn** [comment]
Je kunt een marge nemen van 35 pg/ml tot aan 90 pg/ml voor de BNP waarden :)
